core/domain/state_domain.py

```diff
@@ -3327,6 +3420,7 @@ def create_default_state(
     @classmethod
     def convert_html_fields_in_state(
```

Comment:
Method 1:
Updating the existing `convert_html_fields_in_state`, to also update `image_sizes_in_bytes` attribute.
Similarly, we update image_sizes_in_bytes for all contents of the state (outcome, answer group, hint) etc.